detections/network/inactive_account_have_activity_detected.yml

```diff
@@ -0,0 +1,80 @@
+name: Inactive Account Have Activity Detected
+id: e08aa2f6-7d90-4ab8-af11-da4df38bb7ff
+version: 1
+date: '2024-10-14'
+author: Zaki Zarkasih Al Mustafa
+data_sources:
+- Windows Security 4624
```

Comment:
Should be :
- Windows Event Log Security 4624
- Windows Event Log Security 4625

We use names. which are defined here:

https://github.com/splunk/security_content/blob/develop/data_sources/windows_event_log_security_4624.yml#L1